api/v1alpha1/clusterextension_types.go

```diff
@@ -80,14 +82,15 @@ const SourceTypeCatalog = "Catalog"
 type SourceConfig struct {
```

Comment:
I believe we need to update the CEL validation here to match the changes in https://github.com/operator-framework/catalogd/pull/443/files#diff-d446855f2187e6d61c4719c3202fcb21e4aea9b09e09f9e1d01419f341d8fc9fR233

It looks like Joel had requested this as well: https://github.com/openshift/api/pull/2067/files#r1812412946